generate_grpc.sh

```diff
@@ -11,15 +11,15 @@ if [ ! -e ./vendor_proto/protobuf/src/google/protobuf ]; then
 fi
 
 # controller
-protoc -I pkg/engine/controller/rpc/pb -I vendor_proto/protobuf/src/ pkg/engine/controller/rpc/pb/controller.proto --go_out=plugins=grpc:pkg/engine/controller/rpc/pb
+protoc -I pkg/engine/controller/rpc/pb -I vendor_proto/protobuf/src/ pkg/engine/controller/rpc/pb/controller.proto --go_out=plugins=grpc:$GOPATH/src/
```

Comment:
Can we avoid using GOPATH? There is no guarantee GOPATH exists. With `go mod`, GOPATH can be empty.

Reply:
OK. I modified it to use the relative path. And I added comments in the file `syncagent.proto` to explain the reason.